apps/designer/app/routes/rest/pages.$projectId.ts

```diff
@@ -13,8 +13,8 @@ import { type FetcherData, makeFieldError } from "~/shared/form-utils";
 
 const nonEmptyString = z
   .string({
-    invalid_type_error: "Can't be empty",
-    required_error: "Can't be empty",
+    invalid_type_error: "Can't be empty", // eslint-disable-line
+    required_error: "Can't be empty", // eslint-disable-line
```

Comment:
Not a good idea to disable all rules. What if in the same line we'll break some other rule that we don't want to be disabled?

Same in other places in the diff.